apps/src/templates/VersionHistory.jsx

```diff
@@ -48,11 +48,19 @@ var VersionHistory = React.createClass({
   },
 
   /**
+   * Called when the server responds to a request to restore a previous version.
+   */
+  onRestoreSuccess: function () {
+    location.reload();
+  },
+
+  /**
    * Called when the user chooses a previous version to restore.
    * @param versionId
    */
   onChooseVersion: function (versionId) {
-    location.search += '&version=' + versionId;
+    // TODO: Use Dave's client api when it's finished.
```

Comment:
Please remove this TODO, unless you can make it more specifically point to work we're actually planning to do